examples/example.js

```diff
@@ -22,6 +22,15 @@ fastify
   .post('/', schema, function (req, reply) {
     reply(null, { hello: 'world' })
   })
+  .head('/', schema, function (req, reply) {
+    reply(null, { hello: 'world' })
```

Comment:
Oh yes, sorry copy&past typo. I can fix these things if you want, or only help you on others things!